### PR TITLE
bump-cask-pr: bump language checksums in non-running hardware

### PR DESCRIPTION
**Pull request by cho-m**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

Related to issue seen in Homebrew/homebrew-cask#112638
The Cask language SHAs in non-running hardware (e.g. ARM for Intel users) are not updated via `bump-cask-pr`.

Needs some more testing (and maybe some refactoring) before merge.

---

**EDIT:** Does not deal with issue where `bump-cask-pr` fails when `cask.language` uses a language block with different key and return value (e.g. `en != en-US` like in `openoffice`).

The problem is that `cask.language` returns the "called" output of language block while `cask.languages` returns a list of the keys.

One solution: run same `cask.language` logic (for this approach, it may be good to refactor so to allow `fetch_cask(tmp_cask)` to avoid reloading cask):
```diff
--- a/Library/Homebrew/dev-cmd/bump-cask-pr.rb
+++ b/Library/Homebrew/dev-cmd/bump-cask-pr.rb
@@ -142,9 +142,10 @@ module Homebrew
         end

         cask.languages.each do |language|
-          next if language == cask.language
-
           lang_config = tmp_config.merge(Cask::Config.new(explicit: { languages: [language] }))
+          tmp_cask.config = lang_config
+          next if tmp_cask.language == cask.language
+
           replacement_pairs << fetch_cask(tmp_contents, config: lang_config)
         end
       end
```

### CODE REVIEW

**Review by miccal — APPROVED** [review]

**Comment by CarterRose** [comment]
I need it in btc wallet

**Comment by cho-m** [comment]
Should be ready to review/merge.

Once merged, should hopefully will help avoid separate hardware PRs
- Homebrew/homebrew-cask#121243 
- Homebrew/homebrew-cask#122114

**Review by miccal — APPROVED** [review]

**Comment by cho-m** [comment]
Since I don't have write access, can someone merge the PR? Thanks.

**Pull request merged**
